lib/angular/fieldRadio/fieldRadioController.js

```diff
@@ -0,0 +1,20 @@
+/*global angular:true*/
+
+function FieldRadioController($scope) {
+  var self = this;
+  self.field = $scope.field;
+  self.field.value = (function() {
+    var options = self.field.fieldOptions.definition.options;
+    for (var i = 0; i < options.length; ++i) {
```

Comment:
[ProTip] Check _.find for nicer way of doing the same: https://lodash.com/docs/4.17.4#find